app/decorators/repository_decorator.rb

```diff
@@ -1,19 +1,37 @@
 # frozen_string_literal: true
 
 class RepositoryDecorator < ApplicationDecorator
+  include ActionView::Helpers::NumberHelper
+
   delegate_all
 
   def self.collection_decorator_class
     PaginatingDecorator
   end
 
   def languages
-    return '' unless language
+    return [] unless language
 
-    language.split(',').first(3).to_sentence
+    language.split(', ').first(3).map(&:strip)
   end
 
   def name
     link[/[\w_-]+$/]
   end
+
+  def issues_formatted
+    format_number(model.issues)
+  end
+
+  def stars_formatted
+    format_number(model.stars)
+  end
+
+  private
+
+  def format_number(number)
+    return (number || 0).to_s if (number || 0) <= 999
```

Comment:
Honestly, It looks worse than before. Why not just let `0` be the default?

Reply:
you arr doing `(number || 0)` twice in a single line when setting `0` as default would solve it already.

Reply:
I tested using default zero, but when the number is `nil` the default value is overrided.

Reply:
I thought the `nil` was for when a value is not given. Why would it receive nil from somewhere?

(If a default value is not possible it's better to back how it was before)

Reply:
I solved, just updating the `schema.rb` for `repositories` table